routers/api/v1/org/team.go

```diff
@@ -10,11 +10,16 @@ import (
 	"code.gitea.io/gitea/models"
 	"code.gitea.io/gitea/modules/context"
 	"code.gitea.io/gitea/routers/api/v1/convert"
+	"code.gitea.io/gitea/routers/api/v1/user"
 )
 
 // ListTeams list all the teams of an organization
 func ListTeams(ctx *context.APIContext) {
 	org := ctx.Org.Organization
+	if !org.IsOrgMember(ctx.User.ID) {
```

Comment:
This depends public or private status of org members. If public, then everyone could visit the members.